### ISSUE TITLE
[Feature request] Mobile Browsers Support

### ISSUE THREAD

**Issue by chelouche9** [xenova/transformers.js#53]
When trying to run the models from a mobile browser it fails. 

Mobile browsers have limited storage for webpages (let's assume 100MB since it depends on the browser and smartphone). 
When using smaller models like whisper-tiny (STT) or distilbert-base-uncased (QA) it works.

Any ideas on how can we solve it?

I was thinking about pruning big models and having a less than 100MB version for them like Tensorflow Lite. What do you think?

**Comment by xenova** [comment]
Well, I would say that is up to the user of the library... 😅  It should be their responsibility to choose models that run on their desired platforms.

There are some ways to produce smaller models (e.g., quantization and pruning), but they can only get you so far...

That said, it probably would be good to build up a larger collection of small models (like [facebook/detr-resnet-50](https://huggingface.co/Xenova/transformers.js/tree/main/quantized/facebook/detr-resnet-50/object-detection), which is only 43MB, but can be used for object detection).

If you have some candidates, you can post links to them here

**Comment by xenova** [comment]
Closing issue as there are models which are able to work on mobile devices. For an example, I have tested [Whisper Web](https://huggingface.co/spaces/Xenova/whisper-web) on my phone and it can run the quantized versions (~61MB in size).

Overall, if a user of the library wishes to deploy a version of their app to run in mobile browsers, it is their responsibility to choose models which can run in a mobile browser.

Alternatively, some users have found ways to get Transformers.js running natively via react-native. For more information, see [this PR](https://github.com/xenova/transformers.js/pull/118).
